app/views/layouts/application.html.erb

```diff
@@ -22,8 +22,8 @@
   </div>
 
   <footer>
-    <span class="stateface stateface-replace stateface-mi">Michigan</span>
-    <span>Lovingly crafted in Michigan.<br>&copy; <%= Time.now.year %> <%= link_to '[i] Collecitve Idea', 'http://collectiveidea.com/' %>.</span>
+    <%= image_tag "MI.svg", alt: "" %>
```

Comment:
We should make this an inline svg… probably. But that was outside the timebox I set for this.